ros_gz_sim/package.xml

```diff
@@ -25,6 +25,9 @@
   <depend>rclcpp_components</depend>
   <depend>rcpputils</depend>
   <depend>std_msgs</depend>
+  <depend>builtin_interfaces</depend>
+  <depend>geometry_msgs</depend>
+  <depend>ros_gz_interfaces</depend>
```

Comment:
include [cli11](https://github.com/ros/rosdistro/blob/master/rosdep/base.yaml#L614) as a `<depend>`. You should also include this in the CMakeLists.txt